core/src/main/java/org/fao/geonet/kernel/mef/Importer.java

```diff
@@ -547,11 +544,13 @@ public static void importRecord(String uuid, MEFLib.UuidAction uuidAction, List<
         if (metadataExist && uuidAction == MEFLib.UuidAction.NOTHING) {
             throw new UnAuthorizedException("Record already exists. Change the import mode to overwrite or generating a new UUID.", null);
         } else if (metadataExist && uuidAction == MEFLib.UuidAction.OVERWRITE) {
+            String recordToUpdateId = metadataUtils.getMetadataId(uuid);
+
             if (isMdWorkflowEnable) {
-                throw new UnAuthorizedException("Overwrite mode is not allowed when workflow is enabled. Use the metadata editor.", null);
+                // If there is a working copy get its id otherwise create a new one and get its id
+                recordToUpdateId = metadataUtils.startEditingSession(context, recordToUpdateId).toString();
```

Comment:
Should this be

```java
if (isMdWorkflowEnable) {
   recordToUpdateId = metadataUtils.startEditingSession(context, uuid);
} else {
   recordToUpdateId = metadataUtils.getMetadataId(uuid);
}
```
So that do don't get the id twice?

Reply:
Sorry - I guess startEditingSession required a metadata id.  I'm not sure why startEditingSession accepts a string instead of integer?